File: python/020.py
#!/usr/bin/env python

import argparse
from math import sqrt

def main():
  """ this computes projecteuler.net problem 020 """
  parser = argparse.ArgumentParser()
  parser.add_argument("last", type=int, help="factorial to compute and sum digits of")

  args = parser.parse_args()
  
  factorial = 1
  for i in range(args.last):
    factorial *= i+1

  fstring = str(factorial)
  flen = len(fstring) 
        
  fsum = 0
  for i in range(flen): 
    fsum += int(fstring[i])

  print "SUM is",fsum

if __name__ == "__main__":
    main()

File: javascript/020.js
"use strict"

let last = 100;
let factorial = 1n;

for(let i=0; i < last; i++)
  factorial *= BigInt(i+1);

let fstring = factorial.toString();
let flen = fstring.length;
      
let fsum = 0;
for(let i=0; i < flen; i++)
  fsum += Number(fstring[i]);

console.log(fsum);
